### PR TITLE
✨ FIRST_TIME composition api

### LINKED ISSUE #5026: Rewrite FIRST_TIME Dev hacks
- remove `nuxt-property-decorator`
- vue composition api way
- script setup
- remove old SubQuery links
- https://github.com/kodadot/nft-gallery/blob/main/FIRST_TIME.md

### PR DESCRIPTION
**Pull request by roiLeo**
## PR Type

- [x] Refactoring

## Context

- [x] Closes #5026 

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

### CODE REVIEW

File: FIRST_TIME.md [hunk 1]
```diff
@@ -116,129 +101,86 @@ pnpm install
...
-      })
-
-      this.nft = data.nFTEntity
-      console.log('nft', this.nft)
-    }
-  }
+<script lang="ts" setup>
+const route = useRoute()
+const { data: nft } = useGraphql({
+  queryName: 'nftByIdMinimal',
```
Comment (roiLeo) [inline FIRST_TIME.md:123]:
```suggestion
  queryName: 'nftById',
```

File: FIRST_TIME.md [hunk 2]
```diff
@@ -116,129 +101,86 @@ pnpm install
 
 **1. How can I read some data from the GraphQL?**
 
-Every `.graphql` file is located in the `src/queries/`.
+Every `.graphql` file is located in the `queries/`.
 
 ```graphql
 query nftByIdMinimal($id: String!) {
-  nFTEntity(id: $id) {
+  nft: nFTEntity(id: $id) {
```
Comment (vikiival) [inline FIRST_TIME.md:108]:
```suggestion
  nft: nftEntityById(id: $id) {
```

File: FIRST_TIME.md [hunk 3]
```diff
@@ -116,129 +101,86 @@ pnpm install
...
-
-    async created() {
-      this.subscribe(
-        api.query.uniques.asset,
-        [this.collectionId, this.id],
-        (nft: any) => console.log(nft) // callback which returns the data
-      )
-    }
-  }
+console.log(nft)
```
Comment (vikiival) [inline FIRST_TIME.md:142]: use consola

File: FIRST_TIME.md [hunk 4]
```diff
@@ -116,129 +101,86 @@ pnpm install
...
+  })
+}
 </script>
 ```
 
-**5. How can I test Kodadot without spending KSM?**
+**4. How can I test Kodadot without spending KSM?**
 
+For Kusama:
 [You can obtain some Westend (WND)](https://matrix.to/#/#westend_faucet:matrix.org)
```
Comment (vikiival) [inline FIRST_TIME.md:177]: remove

File: FIRST_TIME.md [hunk 5]
```diff
@@ -116,129 +101,86 @@ pnpm install
...
 
+For Basilisk (Rococo):
+[You can obtain some KSM & BSX](https://discord.com/channels/840514076538830888/881839877140930630)
+
 You can change the network in the navbar.
-Currently supported networks are `Kusama, Westend, statemine, westmint`.
+Currently supported networks are `Basilisk, Kusama, Westend, statemine, westmint`.
```
Comment (vikiival) [inline FIRST_TIME.md:183]:
```suggestion
Currently supported networks are `Basilisk`, `Basilisk-Rococo` and `Kusama`.
EVM chains such as `MoonBeam` and `MoonRiver` are in read-only mode.

```

File: FIRST_TIME.md [hunk 6]
```diff
@@ -249,10 +191,9 @@ In case you are using Apple M1, we have a [tutorial for that 🍏 ](https://viki
 Current Indexers we have/use:
 
 - SubSquid
+  - Basilisk: [snek](https://github.com/kodadot/snek)
   - Kusama: [rubick](https://github.com/kodadot/rubick)
-- SubQuery
-  - Kusama: [magick](https://github.com/vikiival/magick)
-  - Statemine (Unique NFT pallet): [unique](https://github.com/kodadot/unique)
+  - Moonsama: [click](https://github.com/kodadot/click)
```
Comment (vikiival) [inline FIRST_TIME.md:196]:
```suggestion
  - MoonRiver: [click](https://github.com/kodadot/click)
```

**Comment by vikiival** [comment]
pay 20 usd

**Pull request merged**